samples/go/csv/csv-import/perf_test.go

```diff
@@ -50,29 +46,57 @@ func (s *perfSuite) Test01ImportSfCrimeBlobFromTestdata() {
 	defer s.closeGlob(raw)
```

Comment:
I might rename this test as well, like `Test01ImportSfCrimeBlobOmitCommit`, so that the test results don't appear to change.